modules/oe_theme_helper/tests/src/Kernel/Plugin/Field/FieldFormatter/AddressInlineFormatterTest.php

```diff
@@ -36,11 +36,12 @@ public function testInlineFormatterAddress() {
       'country_code' => 'BE',
       'locality' => 'Brussels',
```

Comment:
It would be nice to reflect in a test that the city also doesn't have anymore double escaping.